### PR TITLE
style(ndm): Fix linter in server/server.go

### PR DESCRIPTION
**Pull request by rbo13**
Fixed the linter in `server/server.go`

Signed-off-by: Richard Burk Orofeo chardy.orofeo@yahoo.com.ph

reference issue: #91

### CODE REVIEW

**Comment by rbo13** [comment]
How to resolve this failing from codecoverage?

**Comment by rbo13** [comment]
Thanks @shovanmaity 👍 will do it :)

**Comment by rbo13** [comment]
Hi @shovanmaity, I got an error from travis this time. I got no idea what happen. Anyway you can guide me?

**Comment by shovanmaity** [comment]
> Hi @shovanmaity, I got an error from travis this time. I got no idea what happen. Anyway you can guide me?

@rbo13 travis was unable to run integration tests as it was not able to bring up minikube. I have restarted the build.

File: pkg/metrics/common.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ import (
 )
 
 var (
+	// StartingTime represents
+	// the starting of time
```
Comment (sagarkrsd) [inline pkg/metrics/common.go]: @rbo13 We can have this comment in one line only rather than having it in two lines since its not a very big comment !

Comment (sagarkrsd) [reply pkg/metrics/common.go]: And also the comment should be more specific like whose starting time it represents !

**Comment by rbo13** [comment]
Okay @shovanmaity, will do it first. I'm sorry if I cannot reply directly, having a hard time finding an internet connection in my place.

**Comment by shovanmaity** [comment]
> Okay @shovanmaity, will do it first. I'm sorry if I cannot reply directly, having a hard time finding an internet connection in my place.

no problem @rbo13 , I have merged pr #128 you can resolve conflicts and update this pr now.

**Comment by rbo13** [comment]
Hi @shovanmaity CC @sagarkrsd, I've already updated this PR :) please check it :) thank you so much for giving me the chance to work on this project 👍

File: pkg/server/server.go [hunk 2]
```diff
@@ -34,6 +35,10 @@ func init() {
 	prometheus.MustRegister(metrics.Uptime)
 }
 
+// StartHttpServer boots up the server
```
Comment (shovanmaity) [inline pkg/server/server.go]: One last review make it 2 line

Comment (rbo13) [reply pkg/server/server.go]: Sorry for the late reply, will do it @shovanmaity 👍 thanks

**Comment by rbo13** [comment]
Updated my PR @shovanmaity 👍

File: pkg/server/server.go (hunk 2 above)
Comment (shovanmaity) [reply pkg/server/server.go]: @rbo13 there was a misunderstanding. I wanted to say convert 5 lines of comments to 2 lines of comments. :slightly_frowning_face:

Comment (rbo13) [reply pkg/server/server.go]: Ooops, my bad :( will change my PR now. Thanks for clarifying

**Comment by shovanmaity** [comment]
> I've updated my PR @shovanmaity

Thanks @rbo13

**Pull request merged**